src/components/Pages/ManageWiki/Features/FederatedProperties.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+  <v-form ref="form" v-model="valid">
+    <v-card color="#fad1d0">
+      <v-card-title>Federated properties</v-card-title>
+      <v-card-text>
+      Federated properties allows you to use the properties of Wikidata on your Wikibase instance.<br>
+      If you turn this on, and confirm the warning, any existing properties on your wikibase will no longer be accessible!
+      </v-card-text>
+      <v-card-actions>
+        <v-btn @click="showModal" color="red">Enable Federated Properties</v-btn>
+      </v-card-actions>
+    </v-card>
+    <v-dialog v-model="dialog" width="500" v-if="dialog">
+      <v-card>
+        <v-toolbar flat dark color="primary">
+            <v-toolbar-title>Change Federated properties settings</v-toolbar-title>
+            <v-spacer></v-spacer>
+            <v-btn icon dark @click="hideModal">
+              <v-icon>close</v-icon>
+            </v-btn>
+          </v-toolbar>
+        <br />
+        <v-card-text>If you turn this on, and confirm the warning, any existing properties on your wikibase will no longer be accessible!</v-card-text>
+        <v-card-text>
+          <v-text-field
+            v-model="userConfirmationInput"
```

Comment:
Getting some console spam for `Property or method "userConfirmationInput" is not defined on the instance but referenced during render.`. This isn't used anyway so we can just delete it.

```suggestion
```

Reply:
done!